apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml

```diff
@@ -449,47 +454,43 @@ paths:
-                            amount:
-                              type: integer
-                              format: int256
-                            subunit_to_unit:
-                              type: integer
-                              format: int256
-                            pegged_currency:
-                              type: string
-                              format: binary
-                            pegged_amount:
-                              type: integer
-                              format: int256
-                            pegged_subunit_to_unit:
-                              type: integer
-                              format: int256
-                            updated_at:
-                              type: string
-                              format: date-time
+                        type: object
+                        additionalProperties:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              currency:
+                                type: string
+                                format: binary
+                              amount:
+                                type: integer
+                                format: int256
+                              subunit_to_unit:
+                                type: integer
+                                format: int256
+                              pegged_currency:
+                                type: string
+                                format: binary
+                              pegged_amount:
+                                type: integer
+                                format: int256
+                              pegged_subunit_to_unit:
+                                type: integer
+                                format: int256
+                              updated_at:
+                                type: string
+                                format: date-time
                     example:
-                      data:
+                      '1':
                         - currency: '0x0000000000000000000000000000000000000000'
                           amount: 220000000000000
                           subunit_to_unit: 1000000000000000000
                           pegged_currency: USD
                           pegged_amount: 4
                           pegged_subunit_to_unit: 100
                           updated_at: '2019-01-01T10:10:10+00:00'
-                        - currency: '0xd26114cd6EE289AccF82350c8d8487fedB8A0C07'
-                          amount: 39260000000000000
-                          subunit_to_unit: 1000000000000000000
-                          pegged_currency: USD
-                          pegged_amount: 4
-                          pegged_subunit_to_unit: 100
-                          updated_at: '2019-01-01T10:10:10+00:00'
+                      data: null
```

Comment:
I'm not sure, I'm having trouble parsing the yamls indentation, but `data: null` seems to not fit here. Isn't `data` the root key holding a value with an object keyed with `'1': ..., '2': ...`, which in turns holds arrays as values?

comparing this with: https://github.com/omisego/elixir-omg/pull/1219/files#diff-44f1ab34a5d784c25fcdf593b3a3e31fR65